src/momento/aio/_scs_grpc_manager.py

```diff
@@ -37,18 +36,30 @@ class _DataGrpcManager:
     version = pkg_resources.get_distribution("momento").version
 
     def __init__(self, auth_token: str, endpoint: str):
-        headers = [
-            Header("authorization", auth_token),
-            Header("agent", f"python:{_ControlGrpcManager.version}"),
-        ]
         self._secure_channel = grpc.aio.secure_channel(
             target=endpoint,
             credentials=grpc.ssl_channel_credentials(),
-            interceptors=[AddHeaderClientInterceptor(headers)],
+            interceptors=_interceptors(auth_token)
         )
 
     async def close(self) -> None:
         await self._secure_channel.close()
 
     def async_stub(self) -> cache_client.ScsStub:
         return cache_client.ScsStub(self._secure_channel)
+
+
+def _interceptors(auth_token: str) -> List[grpc.aio.ClientInterceptor]:
+    headers = [
+        Header("authorization", auth_token),
+        Header("agent", f"python:{_ControlGrpcManager.version}"),
+    ]
+    interceptors = [
+        AddHeaderClientInterceptor(headers),
+        *get_retry_interceptor_if_enabled()
+    ]
+    for interceptor in interceptors:
+        print(f"CHECKING INTERCEPTOR: {interceptor} ({interceptor.__class__})")
+        print(f"IS UNARYUNARY: {isinstance(interceptor, grpc.UnaryUnaryClientInterceptor)}")
+        print(f"IS AIO UNARYUNARY: {isinstance(interceptor, grpc.aio.UnaryUnaryClientInterceptor)}")
```

Comment:
looks like maybe you forgot to rm some debuggy stuff

Reply:
ty